extensions/notebook/src/jupyter/serverInstance.ts

```diff
@@ -339,12 +338,14 @@ export class PerFolderServerInstance implements IServerInstance {
 		//                http://localhost:8888/?token=f5ee846e9bd61c3a8d835ecd9b965591511a331417b997b7
 		let dataString = data.toString();
 		let urlMatch = dataString.match(/\[C[\s\S]+ {8}(.+:(\d+)\/.*)$/m);
-
 		if (urlMatch) {
 			// Legacy case: manually parse token info if no token/port were passed
 			return [vscode.Uri.parse(urlMatch[1]), urlMatch[2]];
-		} else if (this._uri && dataString.indexOf(JupyterStartedMessage) > -1) {
+		} else if (this._uri && dataString.match(/jupyter notebook .*is running at:/im)) {
```

Comment:
Is this message ever going to be localized?

Reply:
It's not localized in Jupyter: https://github.com/jupyter/notebook/blob/f5d6995089c044a320089f0021273c26930fe88a/notebook/notebookapp.py#L2071